docs/ref/contrib/postgres/indexes.txt

```diff
@@ -34,3 +34,17 @@ available from the ``django.contrib.postgres.indexes`` module.
     PostgreSQL. You can install it using the
     :class:`~django.contrib.postgres.operations.BtreeGinExtension` migration
     operation.
+
+    Set the ``fastupdate`` parameter to ``False`` to disable the `GIN Fast
+    Update Technique`_ that's enabled by default in PostgreSQL.
+
+    Provide an integer number of bytes to the gin_pending_list_limit_ parameter
```

Comment:
It's only required for multi word links, http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets